website/docs/reference/operators/query/logical-operators/and.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 1
+description: Combines multiple query expressions and outputs documents that match all expressions.
+---
+
+# `$and`
+
+import CodeBlock from '@theme/CodeBlock';
+import Dataset from '!!raw-loader!./logical-operators/2-insert-dataset.request.js';
+import AndOperatorRequest from '!!raw-loader!./logical-operators/3-and-operator.request.js';
+import AndOperatorResponse from '!!raw-loader!./logical-operators/3-and-operator.response.js';
+
+The `$and` operator combines multiple query expressions and returns documents that match all the expressions.
+
+The operator accepts an array of query expressions as shown below, and all expressions must be satisfied for a document to be included in the result set.
+
+```js
+db.runCommand({
+  find: '<collection_name>',
+  filter: {
+    $and: [
+      { <query_expression1> },
+      { <query_expression2> }
+    ]
+  }
+})
+```
+
+A query expression can be any valid query such as:
+
+- `{ <field>: <value> }`
```

Comment:
Could we use ```json format where syntax highlighting is used? Or possibly `js`? I think we should use syntax highlighting as it's come across easier read.

Reply:
We should not use JSON syntax highlighting here since it's not really talking about an operation but a query expression which can be written in different ways. Its main use is to explain what's already written within the syntax.